src/Mailer/Ownership/RecipientStrategy/SendToRequesterOfOwnership.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailer\Ownership\RecipientStrategy;
+
+use CultuurNet\UDB3\Ownership\Repositories\OwnershipItem;
+use CultuurNet\UDB3\User\UserIdentityDetails;
+use CultuurNet\UDB3\User\UserIdentityResolver;
+use Psr\Log\LoggerInterface;
+
+class SendToRequesterOfOwnership implements RecipientStrategy
+{
+    private UserIdentityResolver $identityResolver;
+    private LoggerInterface $logger;
+
+    public function __construct(UserIdentityResolver $identityResolver, LoggerInterface $logger)
+    {
+        $this->identityResolver = $identityResolver;
+        $this->logger = $logger;
+    }
+
+    /** @return UserIdentityDetails[] */
+    public function getRecipients(OwnershipItem $item): array
+    {
+        $ownerDetails = $this->identityResolver->getUserById($item->getOwnerId());
```

Comment:
The call `getOwnerId` gets the owner not the requester. The requester is the user who asks for the ownership, but he/she can ask that for another person. That is basically what happens if an owner of an organizer goes to the ownership page and adds a new owner.